dockerfile/anaconda-ci/run-build-and-arg

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -eux
+
+# Avoid clobbering the host's checkout, so work from a copy
+cp -r /anaconda /tmp/
+cd /tmp/anaconda
+
+./autogen.sh
+./configure
+make
+
+# always copy the results back to the host
+copy_logs() {
+    mkdir -p /anaconda/test-logs/
+    # clean up from previous run
+    rm -rf /anaconda/test-logs/*
+    find -name '*.log' -exec cp '{}' /anaconda/test-logs/ \;
```

Comment:
Btw. this will find logs in `/tmp/anaconda/test-logs` that will override everything else, so I never get a new version of the logs.

Reply:
Sorry, I'm afraid I don't understand what you mean. Can you give a step-by-step description how this goes wrong?

My intention was that each run cleans up the host directory first, to avoid having outdated logs. Otherwise you could have an early failure, and still find the previous run's nosetests.log or so in the log directory, and think "all is well".

Reply:
How to reproduce?

1. Clone the repository.
2. Create an error in the code.
3. Run unit tests for the first time. This step will create the `test-logs` directory on my host.
4. Remove the error from the code.
5. Run unit tests for the second time. The stdout shows that the tests passed, but the `test-logs` directory contains logs from the first run.

What happened in the step 5?

1. The existing `test-logs` directory with logs from the first run is copied to `/tmp/anaconda`.
2. Tests are run for the second time. The `/tmp/anaconda` directory contains new logs, but `/tmp/anaconda/test-logs` still contains logs from the first run.
3. The files in `/anaconda/test-logs/` are removed, but the files in `/tmp/anaconda/test-logs/` still exists.
4. The command `find` copies logs from `/tmp/anaconda` to `/anaconda/test-logs/`. The files are processed in the alphanumeric order, so `/tmp/anaconda/tests` is processed before `/tmp/anaconda/test-logs`.
5. Files copied from `/tmp/anaconda/test-logs` overwrites files copied from `/tmp/anaconda/tests`.
6. The `/anaconda/test-logs/` directory contains logs from the first run.

Reply:
Should be fixed now. PTAL?

Reply:
It is fixed. Thanks!